hedera-node/src/test/java/com/hedera/services/state/merkle/MerkleAccountStateSerdeTest.java

```diff
@@ -0,0 +1,121 @@
+package com.hedera.services.state.merkle;
+
+import com.hedera.services.state.submerkle.EntityId;
+import com.hedera.services.state.submerkle.FcTokenAllowanceId;
+import com.hedera.services.state.submerkle.TokenAssociationMetadata;
+import com.hedera.test.serde.SelfSerializableDataTest;
+import com.hedera.test.utils.SerdeUtils;
+import com.swirlds.common.constructable.ClassConstructorPair;
+import com.swirlds.common.constructable.ConstructableRegistry;
+import com.swirlds.common.constructable.ConstructableRegistryException;
+
+import java.util.Collections;
+import java.util.SplittableRandom;
+
+import static com.hedera.services.state.merkle.MerkleAccountState.RELEASE_0230_VERSION;
+import static com.hedera.test.utils.SerdeUtils.approvedForAllAllowancesFrom;
+import static com.hedera.test.utils.SerdeUtils.byteStringFrom;
+import static com.hedera.test.utils.SerdeUtils.entityIdFrom;
+import static com.hedera.test.utils.SerdeUtils.entityNumPairFrom;
+import static com.hedera.test.utils.SerdeUtils.grantedCryptoAllowancesFrom;
+import static com.hedera.test.utils.SerdeUtils.grantedFungibleAllowancesFrom;
+import static com.hedera.test.utils.SerdeUtils.keyFrom;
+import static com.hedera.test.utils.SerdeUtils.stringFrom;
+import static com.hedera.test.utils.SerdeUtils.unsignedIntFrom;
+import static com.hedera.test.utils.SerdeUtils.unsignedLongFrom;
+import static com.swirlds.common.CommonUtils.unhex;
+
+public class MerkleAccountStateSerdeTest extends SelfSerializableDataTest<MerkleAccountState> {
+	private static SplittableRandom r = new SplittableRandom(4_242_424);
```

Comment:
on second thought, this random generator should be moved into getExpectedObject() ... Otherwise, two calls to getExpectedObject will yield different objects.

Reply:
Consolidated all seeded property generation into a [`SeededPropertySource` helper class](https://github.com/hashgraph/hedera-services/blob/fix-0250-serde-bugs/hedera-node/src/test/java/com/hedera/test/utils/SeededPropertySource.java#L66).